source/config.d

```diff
@@ -212,7 +213,7 @@ class Config
 
         JSONValue root = parseJSON (jsonData);
 
-        workspaceDir = dirName (fname);
+        workspaceDir = dirName (fname).absolutePath;
```

Comment:
If we always access things in the workspace directory as an absolute path, then (at least some of) the thread safety problems with libarchive don't happen.